red-hat-data-services/odh-manifests
# Add role for servingruntime configmap

## How Has This Been Tested?



1. Deploy all the files in the manifest folder
2. Create a new Model Server
3. Check that the ServingRuntime is created with the specification of the `servingruntimes-config` ConfigMap.
4. Now use #806 to fake a regular user.
5. Repeate the same process
6. Ensure regular user with no read options can access the manifest too

You can test that is currently working with a regular user with no read access just running:

```
oc get configmap servingruntimes-config -n redhat-ods-applications
NAME                     DATA   AGE
servingruntimes-config   1      12m
```
